src/averageAges.js

```diff
@@ -37,7 +46,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const ArrMothers = people.filter((person) => {
```

Comment:
```suggestion
  const arrMothers = people.filter((person) => {
```
Don't capitalise the first letter

Reply:
Not fixed